arch/arm/src/armv7-a/arm_vectors.S

```diff
@@ -1096,7 +1098,7 @@ g_intstackbase:
 	.type	g_fiqstackbase, object
 
 g_fiqstackalloc:
-	.skip	((CONFIG_ARCH_INTERRUPTSTACK & ~7) - 4)
+	.skip	(CONFIG_ARCH_INTERRUPTSTACK & ~7)
```

Comment:
Should we keep the `.skip` in the base below?

Reply:
@Ouss4 
I will apply the align up logic too.
Also, I will check the armv8-m and so on.

Reply:
I will follow the discussion results for arch/arm/src/armv7-m/gnu/arm_exception.S

Reply:
As a result of the discussion, we decided to keep the commit.

Reply:
Done